rootfs/bin/backup-initial

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+until psql -l -t >/dev/null 2>&1; do sleep 1; done
```

Comment:
This line should be in "if then - fi"  blocks?

(refs https://github.com/deis/postgres/issues/123#issuecomment-234714776)